lib/pfconfig/namespaces/config/Pfmon.pm

```diff
@@ -0,0 +1,86 @@
+package pfconfig::namespaces::config::Pfmon;
+
+=head1 NAME
+
+pfconfig::namespaces::config::Pfmon
+
+=cut
+
+=head1 DESCRIPTION
+
+pfconfig::namespaces::config::Pfmon
+
+This module creates the configuration hash associated to pfmon.conf
+
+=cut
+
+use strict;
+use warnings;
+
+use pfconfig::namespaces::config;
+use pf::file_paths qw($pfmon_config_file $pfmon_default_config_file);
+use pf::util qw(normalize_time);
+use pf::IniFiles;
+use Clone qw(clone);
+
+use base 'pfconfig::namespaces::config';
+
+sub init {
+    my ($self) = @_;
+    $self->{file} = $pfmon_config_file;
+    my $defaults = pf::IniFiles->new(-file => $pfmon_default_config_file);
+    $self->{added_params}{'-import'} = $defaults;
+}
+
+our %TIME_ATTR = (
```

Comment:
This would be better placed in pf::constants::pfmon or somewhere out of this module at least